docs/source/user-guide/sql/sql_status.md

```diff
@@ -34,106 +34,63 @@
 
 ## SQL Support
 
-- [x] Projection
-- [x] Filter (WHERE)
-- [x] Filter post-aggregate (HAVING)
-- [x] Limit
-- [x] Aggregate
-- [x] Common math functions
-- [x] cast
-- [x] try_cast
+- [x] Projection (`SELECT`)
+- [x] Filter (`WHERE`)
+- [x] Filter post-aggregate (`HAVING`)
+- [x] Sorting (`ORDER BY`)
+- [x] Limit (`LIMIT`
+- [x] Aggregate (`GROUP BY`)
+- [x] cast /try_cast
 - [x] [`VALUES` lists](https://www.postgresql.org/docs/current/queries-values.html)
-- Postgres compatible String functions
-  - [x] ascii
-  - [x] bit_length
-  - [x] btrim
-  - [x] char_length
-  - [x] character_length
-  - [x] chr
-  - [x] concat
-  - [x] concat_ws
-  - [x] initcap
-  - [x] left
-  - [x] length
-  - [x] lpad
-  - [x] ltrim
-  - [x] octet_length
-  - [x] regexp_replace
-  - [x] repeat
-  - [x] replace
-  - [x] reverse
-  - [x] right
-  - [x] rpad
-  - [x] rtrim
-  - [x] split_part
-  - [x] starts_with
-  - [x] strpos
-  - [x] substr
-  - [x] to_hex
-  - [x] translate
-  - [x] trim
-- Conditional functions
-  - [x] nullif
-  - [x] case
-  - [x] coalesce
-- Approximation functions
-  - [x] approx_distinct
-  - [x] approx_median
-  - [x] approx_percentile_cont
-  - [x] approx_percentile_cont_with_weight
-- Common date/time functions
-  - [ ] Basic date functions
-  - [ ] Basic time functions
-  - [x] Basic timestamp functions
-    - [x] [to_timestamp](./scalar_functions.md#to_timestamp)
-    - [x] [to_timestamp_millis](./scalar_functions.md#to_timestamp_millis)
-    - [x] [to_timestamp_micros](./scalar_functions.md#to_timestamp_micros)
-    - [x] [to_timestamp_seconds](./scalar_functions.md#to_timestamp_seconds)
-    - [x] [extract](./scalar_functions.md#extract)
-    - [x] [date_part](./scalar_functions.md#date_part)
-- nested functions
-  - [x] Array of columns
+- [x] [String Functions](./scalar_functions.md#string-functions)
+- [x] [Conditional Functions](./scalar_functions.md#conditional-functions)
+- [x] [Time and Date Functions](./scalar_functions.md#time-and-date-functions)
+- [x] [Math Functions](./scalar_functions.md#math-functions)
+- [x] [Aggregate Functions](./aggregate_functions.md) (`SUM`, `MEDIAN`, and many more)
 - [x] Schema Queries
-  - [x] SHOW TABLES
-  - [x] SHOW COLUMNS FROM <table/view>
-  - [x] SHOW CREATE TABLE <view>
-  - [x] information_schema.{tables, columns, views}
-  - [ ] information_schema other views
-- [x] Sorting
-- [ ] Nested types
-- [ ] Lists
+  - [x] `SHOW TABLES`
+  - [x] `SHOW COLUMNS FROM <table/view>`
+  - [x] `SHOW CREATE TABLE <view>`
+  - [x] Basic SQL [Information Schema](./information_schema.md) (`TABLES`, `VIEWS`, `COLUMNS`)
+  - [ ] Full SQL [Information Schema](./information_schema.md) support
+- [x] Support for nested types (`ARRAY`/`LIST` and `STRUCT`)- see [Array Functions](./scalar_functions.md#array-functions)
```

Comment:
I'm not sure whether DF have fully supported it

Reply:
Yes, you are right that the support is only partial - I'll see if I can find some way to make this clearer

Reply:
I think this better reflects the current status:

![Screenshot 2023-06-23 at 12 04 47 PM](https://github.com/apache/arrow-datafusion/assets/490673/a5e91850-8635-4e88-88da-be5878a90e0c)

Reply:
I think good JSON support is very important to get Datafusion more traction in the general community. SQL2023 has some good stuff about this (see [here for a good summary](http://peter.eisentraut.org/blog/2023/04/04/sql-2023-is-finished-here-is-whats-new).).

[This DuckDB page](https://duckdb.org/2022/05/04/friendlier-sql.html) is also a good list of desiderata for us (some of these we implemented already).

I plan to actively talk about/promote Datafusion in various venues once we get into a state where these things "just work".

Reply:
> I think good JSON support is very important for Datafusion to get more traction in the general community. SQL2023 has some good stuff about this (see here for a good summary.).

I agree.

> While we are on the SQL support/ease-of-use topic, this DuckDB page is also a good list of desiderata for us (some of these we implemented already).

 It would be great to file tickets about these features -- I have found clearly written tickets with a "good first issue" often attracts contributions. If you have a chance to file the tickets that would be awesome, otherwise I will try and find time to do so


> I plan to actively talk about/promote Datafusion in various venues once we get into a state where these things "just work".

I think it is a balance -- part of the way we grow the DataFusion community (to get the resources to make it better) is to talk about it publically.

Reply:
We will file tickets for these 👍